README.md

```diff
@@ -51,7 +51,8 @@ If you want to uninstall algorithms, it is as simple as:
     $ pip3 uninstall -y algorithms
 
 ## List of Implementations
-
+- [Greedy](algorithms/arrays)
```

Comment:
The elements are listed in alphabetical order. Could you add it below dp?

Reply:
ok no issues I will do it